Index: app/src/main/java/edu/ucsd/cse110/lab5_room/internal/Constants.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/edu/ucsd/cse110/lab5_room/internal/Constants.java b/app/src/main/java/edu/ucsd/cse110/lab5_room/internal/Constants.java
--- a/app/src/main/java/edu/ucsd/cse110/lab5_room/internal/Constants.java	(revision 9f794ea197d52f28b3148caa1d14b9e25d375b0f)
+++ b/app/src/main/java/edu/ucsd/cse110/lab5_room/internal/Constants.java	(date 1644814085549)
@@ -1,5 +1,7 @@
 package edu.ucsd.cse110.lab5_room.internal;
 
+import com.google.gson.Gson;
+
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
@@ -7,15 +9,15 @@
 public final class Constants {
     private Constants() {}
 
-    public static String INTENT_LOGGED_IN = "logged_in";
-    public static Set<Character> CHARSET_ALPHA_LATIN = new HashSet<>(Arrays.asList(
+    public static final String INTENT_LOGGED_IN = "logged_in";
+    public static final Set<Character> CHARSET_ALPHA_LATIN = new HashSet<>(Arrays.asList(
             'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
             'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
             'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
             'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z',
             ' ', '-', '\''
     ));
-    public static Set<Character> CHARSET_URL = new HashSet<>(Arrays.asList(
+    public static final Set<Character> CHARSET_URL = new HashSet<>(Arrays.asList(
             'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
             'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
             'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
@@ -24,4 +26,15 @@
             '/', ':', '?', '%', '&', '[', ']', '!', '$', '\'', '(', ')', '*',
             '+', ',', ';', '='
     ));
+
+    public static final Gson GSON = new Gson();
+
+    // remove or add users to discover list
+    public static final int ACTION_ADD = 0;
+    public static final int ACTION_REMOVE = 1;
+
+    public static final String USER_NAME    = "name";
+    public static final String USER_PFP     = "profileURL";
+    public static final String USER_COURSES = "courses";
+    public static final String USER_ID      = "studentId";
 }
